tests/tests/cpld_bmark_wave

```diff
@@ -58,7 +57,7 @@ export LIST_FILES="phyf024.tile1.nc \
                    RESTART/MOM.res_2.nc \
                    RESTART/MOM.res_3.nc \
                    RESTART/iced.2013-04-02-00000.nc"
-                 # ufs.s2s.cpl.r.2013-04-02-00000.nc"
+                   #RESTART/ufs.cpld.cpl.r.2013-04-02-00000.nc"
```

Comment:
Maybe it's good to create an issue for the nccmp comparison of this file?

Reply:
I created issue #242